### DISCUSSION TITLE
Calculate polygon properties : Centroid, Area, Inertia ...

### DISCUSSION THREAD

**Discussion by hdahmani1** [NetTopologySuite/NetTopologySuite#637] [Q&A]
Hi, 
Can you please confirm (or not) if we can calculate geometry properties (centroid, area, inertia ...) ?
Thank you

**Comment by DGuidi** [comment]
heres' the a list of the available properties, from the [nts docs](https://nettopologysuite.github.io/NetTopologySuite/api/NetTopologySuite.Geometries.Geometry.html)
no idea of what inertia is

**Reply by hdahmani1** [reply]
Thank you for your answer. (Inertia = Second moment of area)

**Reply by hdahmani1** [reply]
Is there a way to add some geometric characteristics to existing class 'Polygon'?

**Reply by DGuidi** [reply]
hard to say without being more specific: basically you can extend the polygon class or attach properties via the user data property, but either way you lose this information after topologycal operations

**Reply by FObermaier** [reply]
You can write extension methods for the polygon class. That is not exactly a property but it should do.
